libyui-qt/pkgconfig/libyui-qt.pc.in

```diff
@@ -0,0 +1,23 @@
+#
+# Pkg-Config file for libyui-qt
+#
+# Generated by CMake from libyui-qt.pc.in
+#
+
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${exec_prefix}/include/yui/qt
+plugindir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@/yui
+
+soversion_major=@SONAME_MAJOR@
+soversion_minor=@SONAME_MINOR@
+soversion_patch=@SONAME_PATCH@
+soversion=@SONAME@
+
+Name: libyui-qt
+Version: @VERSION@
+Description: libyui-qt - Qt plug-in for libyui
+Libs: -L${plugindir} -lyui-qt
+Cflags: -I${includedir}
```

Comment:
Cflags and include dir should follow the changes added by @mvidner in libyui.pc PR to allow including files correctly: something like:
`includedir=${exec_prefix}/include`
and 
`Cflags: -I${includedir}/yui -I${includedir}/yui/qt -I${includedir}`

Reply:
Right, good idea; then we are flexible how an extension can actually use the include files:

```C++
#include <yui/qt/YQUI.h>
#include <qt/YQUI.h>
#include <YQUI.h>
```

I'll change it accordingly.